### ISSUE TITLE
Draggable Bookmarklets

### ISSUE THREAD

**Issue by mrtysn** [DevBubba/Bookmarklets#2]
Nice collection!

I don't know whether this is common or not, but I am used to having draggable bookmarklets. I wanted to try the bookmarklets on the website but dragging them to the bookmarks bar on Google Chrome creates a bookmark with a direct link to the bookmarklet's source on _raw.githubusercontent.com_.

Draggable Bookmarklet Examples:
-  The [Waterize bookmarklet](https://watercss.kognise.dev/). You can find the line that says: "Just drag this link to your bookmarks bar".
- A tool that creates draggable bookmarklets: https://chriszarate.github.io/bookmarkleter/.

I think your scripts are already in the correct format since they work as expected if you create the bookmarklet manually through the bookmark creation UI. 

I believe the drag and drop behavior is created like this:
`<a href="bookmarklet_javascript_code">Bookmark Title</a>`

What do you think about making this happen? It would greatly improve the usability of your collection in my opinion.

**Comment by Kbeny** [comment]
On chrome copy the code and right click on the bookmark bar the press paste and enjoy hope it works for you too

**Comment by mrtysn** [comment]
yes @Kbeny , you can manually create bookmarks that way. However, the way I'm describing is a little more convenient as it's just a drag and drop operation.

**Comment by Kbeny** [comment]
I mean sure that's convenient @mrtysn but this works and it may be easier for him

**Comment by DevBubba** [comment]
I would love to find an easy method for draggable bookmarklets and I have a few more fixes in mind but I have yet have to modify or update this repo. **React or let me know on this post** if I should add draggable bookmarklets and **finally update the repo after 2 years!**
